docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -72,6 +72,33 @@ Dependency search provides the following filters, which correspond to the data p
 ![Screenshot of dependency search with query](/img/SSC-DepSearch-Query.png#md-width)
 _**Figure**. Dependency search page with sample search query._
 
+## Dependency graphs (beta)
+
+:::info
+To participate in this beta, reach out to [sales@semgrep.com](mailto:sales@semgrep.com).
+:::
+
+Dependency graphs allow you to view dependency paths for all transitive dependencies, up to seven layers deep, introduced in a project. With this information, you can understand:
+
+- How a transitive dependency was introduced
+- How deep the transitive dependency is nested in the dependency tree. The dependency tree reflects your project that Semgrep generates
+
+Semgrep generates dependency graphs for Java projects that include a `maven_dep_tree.txt` file whenever you invoke a scan using `semgrep ci`.
```

Comment:
Just one more clarification: Semgrep only makes these dependency graphs for customers that are opted in to the beta. No change is necessarily needed to reflect this, just wanted to make sure we were on the same page

Reply:
Yes, that's my understanding as well